src/arrayMethodSort.js

```
@@ -4,8 +4,27 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => (a + '' >= b + '' ? 1 : -1) || 0,
+  ) {
```

There is no validation for the `compareFunction` argument. If a non-function is passed, native sort throws a `TypeError`. You should add a check to throw if `compareFunction` is not undefined and not a function. This is required by checklist item 3.1.